### ISSUE TITLE
Hosts file had all *scdn* matches removed.

### ISSUE THREAD

**Issue by ChromiaCat** [SpotX-Official/SpotX#588]
### 🧐 Have you read the FAQ and viewed similar issues?

- [X] I have read the FAQ
- [X] I have checked older issues, open and closed
- [X] I did a clean install of SpotX as <a href="https://telegra.ph/SpotX-FAQ-09-19#1.-Clean-installation">described in the FAQ</a>

### Сountry of your account

Greece

### What is your Spotify Plan?

Free

### Do you have Spicetify installed?

No

### ℹ Computer information

```markdown
- Spotify: 1.2.13.661.ga588f749
- Windows: 10 22H2 19045.2965
- PowerShell: 5.1.19041.2673
```

### 📝 Description

Basically the function that removes scdn matches from the hosts file will also remove matches that are not related to Spotify.

I know it backs up the file before but is there a way to know which lines it removed and/or ask if we want the changes to be written?

**Comment by amd64fox** [comment]
your backup copy of the hosts file is here

`C:\Windows\System32\drivers\etc\hosts.bak`

send a copy here

**Comment by ChromiaCat** [comment]
> your backup copy of the hosts file is here
> 
> `C:\Windows\System32\drivers\etc\hosts.bak`
> 
> send a copy here

I've already restored the copy and I've asked in the second sentence of the description if one or both solutions are easy to implement in the install script so users know what modifications are about to be done. I wouldn't make a bug issue if this was intended behavior.

**Comment by amd64fox** [comment]
all need to do in this case is improve the regex so that it only removes sdn lines associated with spotify, so earlier I asked you to demonstrate your hosts file that was restored

**Comment by ChromiaCat** [comment]
> all need to do in this case is improve the regex so that it only removes sdn lines associated with spotify, so earlier I asked you to demonstrate your hosts file that was restored

[hosts.txt](https://github.com/SpotX-Official/SpotX/files/15180037/hosts.txt)
This site is where I got them: https://someonewhocares.org/hosts/ipv6

**Comment by amd64fox** [comment]
fixed regex, thanks for the feedback
